SpotifyAPI/SpotifyAPI.ino

```diff
@@ -0,0 +1,233 @@
+        }
+
+        void connectToWiFi() {
+            Serial.println("Connecting to Wifi");
+
+            WiFi.begin(ssid.c_str(), password.c_str());
+            while (WiFi.status() != WL_CONNECTED) {
+                delay(200);
+            }
+
+            Serial.print("Connected to Wifi Network: ");
+            Serial.println(ssid.c_str());
+        }
+
+        String ssid, password;
+};
+
+
+// SETUP & LOOP
+void setup() {
+  Serial.begin(9600);
+  while (!Serial) {
+    ;  // Wait For Serial Port to Connect.
+  }
+  WiFiUtilities wifiCheck("The Force", "thisistheway");
+  wifiCheck.checkWiFiModule();
+  wifiCheck.connectToWiFi();
+
+  
+}
+
+// TODO: Don't want in loop for redeclaring every time
+SpotifyWiFiClient spotifyClient("a9a84f65fc9f47568870f4c0c0185e3a", "7cb7fe064e1844c19e87a2d475573948", "AQAQL12IRdZORwdVF8sJeSiTrSNRVKbv1yZNqUFcfT6-ztpwK1gDjRFQZ0gQn5kBzKVL_3veFiHku1m8aDFYtThJzY9kjj3oX8_juhXTA9bn5_VSPHdqoSDPnsotBaEMkck");
+void loop() {
+
+    if(static_cast<long>(millis()) - spotifyClient.oauthExpiryTime > 0) {
+        spotifyClient.getOAuthToken();
+        spotifyClient.oauthExpiryTime = static_cast<long>(millis()) + 10000;
+    }
+
+    if(spotifyClient.oauthToken.length() > 0 && static_cast<long>(millis()) - spotifyClient.nextAlbumArtRetrievalTime > 0) {
+        Serial.println("Requesting Album Art");
+        spotifyClient.getAlbumArtURL();
+        spotifyClient.nextAlbumArtRetrievalTime = static_cast<long>(millis()) + 50000;
+    }
+
+    if(spotifyClient.albumArtURL.length() > 0 && static_cast<long>(millis()) - spotifyClient.nextDisplayUpdate > 0) {
+        Serial.println("Retrieving Album Art");
+        spotifyClient.getAlbumArt();
+        spotifyClient.nextDisplayUpdate = static_cast<long>(millis()) + 10000;
+    }
+
+    while(spotifyClient.SSLClient.available()) {
+        if(!spotifyClient.checkHTTPStatus()) return;
+        spotifyClient.skipHTTPHeaders();
+
+        DynamicJsonDocument doc(16384);
+        DeserializationError error = deserializeJson(doc, spotifyClient.SSLClient);
+        if (error) {
+            Serial.println(error.f_str());
+            spotifyClient.SSLClient.stop();
+            return;
+        }
+
+        spotifyClient.oauthToken = doc["access_token"].as<const char*>();
+        spotifyClient.oauthExpiryTime = (doc["expires_in"].as<long>() * 1000) - (10 * 1000);
+        spotifyClient.SSLClient.stop();
+    }
+
+    while (spotifyClient.SSLClient.available()) {
+        if (!spotifyClient.checkHTTPStatus()) return;
+        spotifyClient.skipHTTPHeaders();
+
+        DynamicJsonDocument doc(16384);
+        DeserializationError error = deserializeJson(doc, spotifyClient.SSLClient);
+        if (error) {
+            Serial.println(error.f_str());
+            spotifyClient.SSLClient.stop();
+            return;
+        }
+
+        spotifyClient.albumArtURL = doc["item"]["album"]["images"][2]["url"].as<const char*>();
+        spotifyClient.SSLClient.stop();
+    }
+
```

Comment:
Where did the third if condition go :(

Reply:
I thought since it was commented out it wasn't done yetttttt. I'll add in!